<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect_server</class>
 <widget class="QDialog" name="connect_server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect to server...</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/png/19.jpg);</string>
  </property>
  <widget class="QLineEdit" name="ipInput">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(230, 234, 234)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(45, 46, 50,0);
color:rgb(230, 234, 234);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Enter Your IP:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>105</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>105</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>105</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>23</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>155</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>80</horstretch>
     <verstretch>30</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>155</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num0">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numDelete">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="num9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>155</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="numBlank">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>25</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>15</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Space</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongMsg">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(45, 46, 50,0);
color:rgb(230, 234, 234);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 13pt &quot;Andale Mono&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectBtn</sender>
   <signal>clicked()</signal>
   <receiver>connect_server</receiver>
   <slot>sendRightConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>connect_server</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connectIP()</slot>
  <slot>sendRightConnect()</slot>
 </slots>
</ui>
